CRM/Report/Form/Case/Demographics.php

```diff
@@ -371,10 +385,6 @@ public function groupBy() {
     $this->_groupBy = CRM_Contact_BAO_Query::getGroupByFromSelectColumns($this->_selectClauses, $groupBy);
   }
 
-  public function orderBy() {
-    $this->_orderBy = "ORDER BY {$this->_aliases['civicrm_contact']}.sort_name, {$this->_aliases['civicrm_contact']}.id, {$this->_aliases['civicrm_case']}.id";
-  }
-
```

Comment:
I felt ordering by sort_name and contact ID was superfluous. Technically, they could result in a different sort order (if you have two contacts with the same sort_name) -- but it's unusual to include contact ID as a sort order option in reports (none of the contact reports include it), so I left it out. If you want it added, I can do so.